core/src/main/java/com/linecorp/armeria/internal/server/FileAggregatedMultipart.java

```diff
@@ -61,15 +64,26 @@ public ListMultimap<String, MultipartFile> files() {
         return files;
     }
 
+    private static boolean shouldSkip(BodyPart bodyPart, List<String> parameters) {
+        final String name = bodyPart.name();
+        assert name != null;
+        return !parameters.isEmpty() && !parameters.contains(name);
+    }
+
     public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(ServiceRequestContext ctx,
                                                                                 HttpRequest req) {
+        return aggregateMultipart(ctx, req, Collections.emptyList());
+    }
+
+    public static CompletableFuture<FileAggregatedMultipart> aggregateMultipart(
+            ServiceRequestContext ctx, HttpRequest req, List<String> parameters) {
         final Path destination = ctx.config().multipartUploadsLocation();
-        return Multipart.from(req).collect(bodyPart -> {
+        return Multipart.from(req)
+                        .filterBodyParts(bodyPart -> !shouldSkip(bodyPart, parameters)).collect(bodyPart -> {
```

Comment:
If we always negate the result of `shouldSkip()`, we could maybe replace `!shouldSkip()` with `shouldHandle()`?